src/cmd/linuxkit/run_vcenter.go

```diff
@@ -191,6 +143,66 @@ func runVcenter(args []string) {
 	}
 }
 
+func vCenterConnect(ctx context.Context, newVM vmConfig) (*govmomi.Client, *object.Datastore, *object.DatacenterFolders, *object.HostSystem, object.NetworkReference, *object.ResourcePool) {
+
+	// Parse URL from string
+	u, err := url.Parse(*newVM.vCenterURL)
+	if err != nil {
+		log.Fatalf("URL can't be parsed, ensure it is https://username:password/<address>/sdk")
```

Comment:
might be helpful to include the err message here?  `log.Fatalf("URL can't be parsed, ensure it is https://username:password/<address>/sdk: %v", err)`